### PR TITLE
Update cypher statement transactional requirement in docs

### PR DESCRIPTION
**Pull request by MishaDemianenko**

### CODE REVIEW

**Comment by MishaDemianenko** [comment]
@craigtaverner @mariascharin please take a look into this small update about cypher < - > transaction requirements.

File: cypher/cypher-docs/src/docs/dev/general/transactions.asciidoc [hunk 1]
```diff
@@ -4,12 +4,10 @@
 Any query that updates the graph will run in a transaction.
 An updating query will always either fully succeed, or not succeed at all.
 
-Cypher will either create a new transaction or run inside an existing one:
+Cypher query should always be executed within transaction.
```
Comment (craigtaverner) [inline cypher/cypher-docs/src/docs/dev/general/transactions.asciidoc]:
```suggestion
The Cypher query should always be executed within a transaction.
```

File: cypher/cypher-docs/src/docs/dev/general/transactions.asciidoc [hunk 2]
```diff
@@ -4,12 +4,10 @@
...
+Every statement is executed in a transaction context, the query will run inside it, and nothing will be persisted to disk until that transaction is successfully committed.
```
Comment (craigtaverner) [inline cypher/cypher-docs/src/docs/dev/general/transactions.asciidoc]:
```suggestion
Every statement is executed within the context of the transaction, and nothing will be persisted to disk until that transaction is successfully committed.
```

File: cypher/cypher-docs/src/docs/dev/general/transactions.asciidoc [hunk 3]
```diff
@@ -4,12 +4,10 @@
...
 
-Cypher will either create a new transaction or run inside an existing one:
+The Cypher query should always be executed within a transaction.
+Every statement is executed within the context of the transaction, and nothing will be persisted to disk until that transaction is successfully committed.
 
-* If no transaction exists in the running context Cypher will create one and commit it once the query finishes.
-* In case there already exists a transaction in the running context, the query will run inside it, and nothing will be persisted to disk until that transaction is successfully committed.
-
-This can be used to have multiple queries be committed as a single transaction:
+Following illustrate sequence of operations that can be used to have multiple queries be committed as a single transaction:
```
Comment (craigtaverner) [inline cypher/cypher-docs/src/docs/dev/general/transactions.asciidoc]:
```suggestion
The following illustrates a sequence of operations that can be used to have multiple queries be committed as a single transaction:
```

**Comment by MishaDemianenko** [comment]
@craigtaverner so merge-merge?)

**Review by craigtaverner — APPROVED** [review]

**Pull request merged**